src/main/java/com/dreammaster/scripts/ScriptThaumicTinkerer.java

```diff
@@ -1706,17 +1706,17 @@ public void loadRecipes() {
                 'c',
                 getModItem(ThaumicTinkerer.ID, "kamiResource", 1, 0, missing),
                 'd',
-                "gemFlawlessDiamond",
+                "foilVoid",
                 'e',
-                getModItem(Thaumcraft.ID, "ItemResource", 1, 7, missing),
+                "foilOsmiridium",
                 'f',
-                "gemFlawlessDiamond",
+                "foilVoid",
```

Comment:
why void foil is called twice.
letter d and f need only one entry
same for the others
(i see many recipes follow this logic but so far i know is this obsolete code)

Reply:
i just followed the way it was alr defined. i can change to reflect more modern applications for sure.